### ISSUE TITLE
Channel Blog / Updates

### ISSUE THREAD

**Issue by mochet** [Joystream/joystream#3320]
I am sure this was discussed at some stage, but I am oblivious to whether it went anywhere or is being implemented (I tried search!)

TLDR:
1. A channel creator can post an update to their channel which is like a blog post. That's it.

It has been asked about here: https://github.com/Joystream/atlas/issues/2191

**Comment by bedeho** [comment]
I don't think this was ever implemented, but a channel subreddit is, but it will be removed.
